packages/example-app-base/src/utilities/router/Router.tsx

```diff
@@ -114,7 +116,7 @@ export class Router extends React.Component<IRouterProps, IRouterState> {
         if (component) {
           const componentChildren = this._resolveRoute(route.props.children || []);
           return React.createElement(component, { key: route.key! }, componentChildren);
-        } else if (getComponent) {
+        } else if (!!getComponent) {
```

Comment:
You could probably revert this and save yourself a merge conflict since I have to make the change in #11677 .

Reply:
I think it will "just work" since this is the same change